IdentityCore/src/MSIDJsonSerializableFactory.h

```diff
@@ -23,20 +23,44 @@
 
 #import <Foundation/Foundation.h>
 
-@class MSIDBrokerOperationRequest;
 @protocol MSIDJsonSerializable;
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface MSIDBrokerOperationRequestFactory : NSObject
+@interface MSIDJsonSerializableFactory : NSObject
```

Comment:
Can you also add some unit tests for this one?

Reply:
done